bindings/python/test/test_mongocrypt.py

```diff
@@ -370,21 +373,46 @@ def close(self):
 
 
 class TestMongoCryptCallback(unittest.TestCase):
+    maxDiff = None
 
     @staticmethod
     def mongo_crypt_opts():
         return MongoCryptOptions({
             'aws': {'accessKeyId': 'example', 'secretAccessKey': 'example'},
             'local': {'key': b'\x00'*96}})
 
+    def test_csfle(self):
+        # Test that we can pick up CSFLE automatically
+        encrypter = AutoEncrypter(MockCallback(), self.mongo_crypt_opts(),
+                                  bypass_encryption=False,
+                                  csfle_required=True,
+                                  )
+        self.addCleanup(encrypter.close)
+        import sys
+        extension = {'win32': "C:\\cygwin\\home\\mci-exec/csfle/bin/mongo_csfle_v1.dll",
```

Comment:
We don't use a `~` here because the behavior of os.expanduser differs on windows on Python 3.8+: https://docs.python.org/3/library/os.path.html#os.path.expanduser

Reply:
Done.